Traceback (most recent call last):
  File "/home/kevin/openshift/matrix_benchmark/exec/mlperf/run_ssd.py", line 607, in <module>
    sys.exit(main())
  File "/home/kevin/openshift/matrix_benchmark/exec/mlperf/run_ssd.py", line 591, in main
    wait_for_mig_reconfiguration(gpu_config)
  File "/home/kevin/openshift/matrix_benchmark/exec/mlperf/run_ssd.py", line 539, in wait_for_mig_reconfiguration
    raise RuntimeError("MIG reconfiguration failed ...")
RuntimeError: MIG reconfiguration failed ...
